aiida_vasp/io/doscar.py

```diff
@@ -1,32 +1,84 @@
 """DOSCAR (VASP format) utilities"""
 import numpy as np
 
-from .parser import BaseParser
+from aiida_vasp.utils.aiida_utils import get_data_class
+from aiida_vasp.io.parser import BaseFileParser
 
 
-class DosParser(BaseParser):
+class DosParser(BaseFileParser):
```

Comment:
Please add unit tests for this class in aiida_vasp/io/tests/test_doscar_io.py

Reply:
Added test for the DosParser